sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml

```diff
@@ -236,6 +234,13 @@
                 </goals>
                 <configuration>
                   <skip>true</skip>
+                  <additionalDependencies>
```

Comment:
This was declared in a second declaration of `maven-javadoc-plugin` in this profile, resulting in two declarations of the plugin which was causing Maven warnings (with the potential of breaking in the future).